deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -15,6 +15,30 @@
 """Creates a GCE instance template for Forseti Security."""
 
 
+def get_patch_search_expression(forseti_version):
+    """Returns a glob expression matching all patches to the version of passed in parameter
+
+    #TODO update in server/forseti-instance-server if update here
```

Comment:
Inside the doc comment, we don't need `#` again.  And make it a proper sentence.
```
TODO: Update in server/forseti-instance-server if update here.
```